corehq/apps/users/templates/users/mobile_workers.html

```diff
@@ -0,0 +1,474 @@
+                                       ng-show="!!worker.editUrl">
+                                        <i class="fa fa-user"></i><strong> {{ worker.username }}</strong>
+                                    </a>
+                                </td>
+                                <td class="{{ worker.creationStatus }}">{{ worker.first_name }}</td>
+                                <td class="{{ worker.creationStatus }}">{{ worker.last_name }}</td>
+                                <td class="{{ worker.creationStatus }}">
+                                    <div ng-show="worker.creationStatus==='pending'">
+                                        <i class="fa fa-circle-o-notch fa-spin"></i>
+                                        {% trans 'Pending...' %}
+                                    </div>
+                                    <div ng-show="worker.creationStatus==='success'">
+                                        <span class="text-success">
+                                            <i class="fa fa-check"></i> {% trans 'NEW' %}
+                                        </span>
+                                    </div>
+                                    <div ng-show="worker.creationStatus==='warning'">
+                                        <span class="text-warning">
+                                            <i class="fa fa-exclamation-triangle"></i>
+                                            {% trans "ERROR" %}
+                                        </span>
+                                        <button type="button"
+                                                class="btn btn-default"
+                                                data-toggle="modal"
+                                                data-target="#newMobileWorkerModal"
+                                                ng-click="initializeMobileWorker(worker); usernameAvailabilityStatus = USERNAME_STATUS.AVAILABLE; usernameStatusMessage = 'Username is available.'">
+                                            {% trans 'Retry' %}
+                                        </button>
+                                    </div>
+
+                                </td>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+            {% endangularjs %}
+        </div>
+        <div class="panel panel-default"
+             ng-controller="PaginatedListController"
+             ng-cloak>
+            <div class="panel-heading">
+                <h3 class="panel-title">{% trans 'Mobile Workers' %}</h3>
+            </div>
+            <div class="panel-body">
+                <div class="row">
+                    <div class="col-sm-4">
+                        <div class="input-group">
+                            <input type="text"
+                                   class="form-control"
+                                   ng-keypress="updateQuery($event)"
```

Comment:
seems like this should be on submit? keycode 13 is one of the keycodes that is pretty standard across all machines so i think it's probably fine